stories/Components/Hero/hero.scss

```diff
@@ -117,14 +149,12 @@
   padding-bottom: $mg-spacing-50;
 }
 
-// Child variant
```

Comment:
Can we please not drop the informative comments.

Reply:
I put the comments back and fixed the other issues